app/controllers/cim_instance_controller.rb

```diff
@@ -147,7 +147,7 @@ def process_show(associations = {})
     else
       whitelisted_key = associations.keys.find { |key| key == @display }
       if whitelisted_key.present?
```

Comment:
As we're no longer trying to placate brakeman, `associations.key?(@display)` should be fine here.